swatch-producer-aws/src/main/resources/application.properties

```diff
@@ -125,8 +125,10 @@ quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultAp
 quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".trust-store-password=${clowder.endpoints.swatch-contracts-service.trust-store-password}
 quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".trust-store-type=${clowder.endpoints.swatch-contracts-service.trust-store-type}
 quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".providers=com.redhat.swatch.resteasy.client.SwatchPskHeaderFilter, com.redhat.swatch.aws.resource.DefaultApiExceptionMapper, com.redhat.swatch.common.resteasy.OffsetDateTimeParamConverterProvider
-com.redhat.swatch.processors.BillableUsageProcessor/lookupAwsUsageContext/Retry/maxRetries=${AWS_USAGE_CONTEXT_LOOKUP_RETRIES}
-com.redhat.swatch.processors.BillableUsageProcessor/send/Retry/maxRetries=${AWS_SEND_RETRIES}
+# Setting up the timeout to 2 minutes instead of 30 seconds (default)
+quarkus.rest-client."com.redhat.swatch.clients.contracts.api.resources.DefaultApi".read-timeout=120000
```

Comment:
This is related to https://github.com/RedHatInsights/rhsm-subscriptions/blob/main/CONTRIBUTING.md#rest-client-default-timeout-is-set-to-30-seconds.
Last week, we merged the port of some of the subscription sync to swatch contracts, so it's expected that the swatch contracts service takes a bit longer: https://github.com/RedHatInsights/rhsm-subscriptions/commit/bc28c01f97218190214c183038540f2f1db810b5